flyteadmin/pkg/manager/impl/project_manager.go

```diff
@@ -54,6 +54,12 @@ func (m *ProjectManager) getDomains() []*admin.Domain {
 	return domains
 }
 
+func (m *ProjectManager) GetDomains(ctx context.Context, request admin.GetDomainRequest) *admin.Domains {
```

Comment:
why not simply has one GetDomains method?

Reply:
merged two method into one